Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "CocoaLumberjack",
+    products: [
```

Comment:
We should add our platform requirements.

Reply:
@ffried 
AppleOS requirements?

Reply:
Yep, see [here](https://github.com/ffried/FFFoundation/blob/a645fe75168926fed06fe8bd67cfebb388603919/Package.swift#L8-L13) for an example.

Reply:
@ffried 
I have strong desire to add `Package-SupportPlatforms.swift` OR `SupportedPlatforms.yaml` and read them in `Package.swift` and in `.podspec`.

Reply:
IMHO this is too much hassle for something that rarely changes.